opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -0,0 +1,151 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from opentelemetry.metrics import (
+    Asynchronous,
+    Counter,
+    Histogram,
+    Instrument,
+    Measurement,
+    Meter,
+    MeterProvider,
+    ObservableCounter,
+    ObservableGauge,
+    ObservableUpDownCounter,
+    UpDownCounter,
+)
+from opentelemetry.util.types import Attributes
+
+
+class Measurement(Measurement):
+    def __init__(self, value, **attributes: Attributes):
+        self._value = value
+        self._attributes = attributes
+        super().__init__(value, **attributes)
+
+    @property
+    def value(self):
+        return self._value
+
+    @property
+    def attributes(self):
+        return self._attributes
+
+
+class _Instrument(Instrument):
+    def __init__(self, name, unit_of_measure="", description=""):
+        super().__init__(
+            name, unit_of_measure=unit_of_measure, description=description
+        )
+        self._name = name
+        self._unit_of_measure = unit_of_measure
+        self._description = description
+
+    @property
+    def name(self):
+        return self._name
+
+    @property
+    def unit_of_measure(self):
```

Comment:
Same as above, just a reminder to rename to `unit`.

Reply:
Fixed